app/views/videos/show.html.haml

```diff
@@ -15,35 +15,31 @@
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form
+      = form_for [@video, Review.new] do |f|
         %fieldset
           .form-group
             %label Rate this video
             .row
               .col-sm-3
-                %select.form-control(name="")
-                  %option(value="5") 5 Stars
-                  %option(value="4") 4 Stars
-                  %option(value="3") 3 Stars
-                  %option(value="2") 2 Stars
-                  %option(value="1") 1 Star
+                =f.select :rating, options_for_select([5, 4, 3, 2, 1].map {|number| [pluralize(number, 'Star')]})
+                
           .form-group
             %label Write Review
             .row
               .col-sm-8
-                %textarea.form-control(name="" rows="6")
-        %fieldset.form-group.actions.clearfix
-          %input(type="submit" value="Submit" class="btn")
-          %a(href="") Cancel
+                =f.text_area :context, row: 6, class: 'span6'
```

Comment:
You can review the new css classes here http://getbootstrap.com/css/#grid
The classes that you were using are from Bootstrap 2.3.2, which you used in course 2.